README.md

```diff
@@ -146,6 +148,23 @@ Another example is the tutorial for the SARS-CoV-2 workflow, which lives alongsi
 Subprojects necessitate a distinct URL, such as https://docs.nextstrain.org/projects/augur as opposed to just https://docs.nextstrain.org.
 By default, this means a distinct set of headings / links in the sidebar navigation, making navigating back to https://docs.nextstrain.org more difficult once you have navigated to a subproject.
 
+## Building the SubProject docs
+
+Typically the documentation for each subproject is build using the same commands as the umbrella docs described above:
+```
+cd <repo>/docs
+make clean # optional
+make html
+```
+This may work in the `docs.nextstrain.org` conda environment, but some subprojects require additional dependencies.
```

Comment:
Hmm, I don't think we should suggest using the docs.nextstrain.org Conda env, even with that caveat. It's never the correct thing to do, even if it appears to work.

Reply:
Resolved in 77f7f74